.gitignore

```diff
@@ -1,30 +1,3 @@
-# Macs sometimes add this file
-.DS_Store
-
-# local node modules
-node_modules
-
-# local distribution files for serving
-/dist
-
-# local env files
-.env.local
-.env.*.local
-
-# log files
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-
-# Editor directories and files
```

Comment:
Everything under # Editer directories and files should be there.
This includes:
.idea
.vscode
*.suo
*.ntvs*
*.njsproj
*.sln
*.sw?